cate/ds/esa_cci_odp.py

```diff
@@ -1460,7 +1461,11 @@ async def _init_file_list(self, monitor: Monitor = Monitor.NONE):
         # Compute the data source's temporal coverage
         for file_rec in file_list:
             if file_rec[1]:
-                file_start_date = datetime.strptime(file_rec[1].split('.')[0], _TIMESTAMP_FORMAT)
+                try:
+                    file_start_date = datetime.strptime(file_rec[1].split('.')[0], _TIMESTAMP_FORMAT)
+                except ValueError:
+                    file_start_date = datetime.fromisoformat(file_rec[1].split('.')[0])
```

Comment:
Also `datetime.fromisoformat` may fail with a `ValueError`! Is this handled somewhere else?

Reply:
Would you expect, e.g.
```
                try:
                    file_start_date = datetime.strptime(file_rec[1].split('.')[0], _TIMESTAMP_FORMAT)
                except ValueError:
                    try:
                         file_start_date = datetime.fromisoformat(file_rec[1].split('.')[0])
                    except ValueError:
                         pass
```
There was no error handling before in this section. I am not sure, what should happen if `ValueError` appears from `datetime.fromisoformat()`